scripts/forrt_contribs/tenzing.py

```diff
@@ -85,12 +85,15 @@ def concatenate_true_columns(row, columns):
 
 merged_data = merged_data.sort_values(by='Surname')
 
+# Strip spaces from 'ORCID iD' in merged data
+merged_data['ORCID iD'] = merged_data['ORCID iD'].str.strip()
+
 # Function to format the full name
 def format_name(row):
     # Extract the first name, middle name initial, and surname
-    first_name = row['First name']
+    first_name = row['First name']].strip() if pd.notna(row['First name']) else ""
```

Comment:
```suggestion
    first_name = row['First name'].strip() if pd.notna(row['First name']) else ""
```

Removed the extra ``]`` in both first_name and surname assignments.